go/kbtest/chat.go

```diff
@@ -159,23 +160,23 @@ func (m TlfMock) PublicCanonicalTLFNameAndID(ctx context.Context, arg keybase1.T
 
 type ChatRemoteMock struct {
 	world     *ChatMockWorld
-	readMsgid map[chat1.ConversationID]chat1.MessageID
+	readMsgid map[string]chat1.MessageID
```

Comment:
I wonder if it's possible to have arrays instead of slices in avdl generated code? i.e. [10]byte instead of []byte. This not only regulates number of bytes for the type, but also make it a value type so we can easily copy it around and use it as map key. Several places could benefit from this, e.g. TopicID, ConversationID, TlfID, etc.

I've had this question before but never brought it up; this seems like a proper applicable scenario hence the comment.